Configuration/AlCa/python/autoCond.py

```diff
@@ -2,60 +2,60 @@
 
     ### NEW KEYS ###
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run1
-    'run1_design'       :   '91X_mcRun1_design_v2',
+    'run1_design'       :   '92X_mcRun1_design_v0',
     # GlobalTag for MC production (pp collisions) with realistic alignment and calibrations for Run1
-    'run1_mc'           :   '91X_mcRun1_realistic_v2',
+    'run1_mc'           :   '92X_mcRun1_realistic_v0',
```

Comment:
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/91X_mcRun1_realistic_v2/92X_mcRun1_realistic_v0
shows that there are no differences between the two GTs
The PFCalibrationRcd should be updated in all GTs